src/main/resources/grammars/Java8ReducedParser.g4

```diff
@@ -281,8 +281,34 @@ inheritanceModifier
     | 'abstract'
     ;
 
+/**
+* @todo #15:30min Classes are generated with only one constructor.
+* Currently, generating multiple constructors can cause a problem
+* where a constructor with the same signature is generated several times.
+* We need to make sure that only unique constructors with different numbers of parameters are generated.
+*/
+
 classBody /* $jsmith-scope */
-    : '{' classBodyDeclaration+ '}' NL
+    : '{' constructorDeclaration classBodyDeclaration+'}' NL
```

Comment:
@qoxrdw Class might be without a constructor. Maybe we can make `constuctorDeclaration` optional? What do you think?